data/archiver/stream.py

```diff
@@ -0,0 +1,172 @@
+from io import BytesIO, StringIO
+import os
+import s3fs
+import hashlib
+import gzip
+import zlib
+import shutil
+from ftplib import FTP, FTP_TLS
+from io import BytesIO
+from memory_profiler import profile
+import tempfile
+import time
+
+from data.archiver.config import AWS_ACCESS_KEY, AWS_SECRET_KEY, ENA_FTP_HOST, ENA_WEBIN_USER, ENA_WEBIN_PWD
+from data.archiver.ftp_uploader import FtpUploader
+
+s3 = s3fs.S3FileSystem(anon=False, key=AWS_ACCESS_KEY, secret=AWS_SECRET_KEY)
+ftp = FTP(ENA_FTP_HOST, ENA_WEBIN_USER, ENA_WEBIN_PWD, timeout=60*60*2) # 2h
+
+
+@profile
```

Comment:
Should we have the profiling code in the production code?
Isn't this part of testing? Or at least, should profiling be turned of and on using some config flag?

Reply:
> Should we have the profiling code in the production code?
No
> Isn't this part of testing? Or at least, should profiling be turned of and on using some config flag?
I left it there to test/profile on dev, but has been removed now.